tests/python_client/testcases/test_utility.py

```diff
@@ -285,7 +285,7 @@ def test_wait_for_loading_collection_not_existed(self):
         self.utility_wrap.wait_for_loading_complete(
             c_name,
             check_task=CheckTasks.err_res,
-            check_items={ct.err_code: 1, ct.err_msg: "can't find collection"})
+            check_items={ct.err_code: 4, ct.err_msg: "collection default:utility_muVBWmM8: collection not found"})
```

Comment:
what does this mean? collection name?

Reply:
Yes. I have modified the code here.

Reply:
Maybe the message default:collectionName more user friendly?  Please track this.